source/CIGameScene.cpp

```diff
@@ -180,6 +180,7 @@ void GameScene::update(float timestep) {
     
     _planet->update(timestep);
     _stardustContainer->update();
+    _stardustContainer->getStardustNode()->update(timestep);
```

Comment:
Could this be changed so that StardustContainer::update takes in timestep and itself calls StardustNode::update? That's the design pattern we've been using for animating the planet